src/OrchardCore.Cms.Web/Properties/launchSettings.json

```diff
@@ -16,12 +16,13 @@
       }
     },
     "web": {
-      "commandName": "web",
+      "commandName": "Project",
```

Comment:
Unwanted changes?

Reply:
in order to run without iis express ....

Reply:
I am not running with IIS, I don't understand the change

Reply:
I am using VS2017 15.6.4 and when I select OrchardCore.CMS.Web as startup project and debug the 'web' profile I get an error "The project doesn't know to run the profile web" 
![capture](https://user-images.githubusercontent.com/9623255/38253299-0393d90a-375f-11e8-8d52-a92353892304.PNG)
Changing command name to "Project" works for me, and I forgot it in the commit